doc/articles/features/working-with-xaml-hot-reload.md

```diff
@@ -85,6 +86,43 @@ Mobile targets are currently using a limited version of XAML Hot Reload and do n
 
 Hot Reload is supported by Visual Studio for WinAppSDK and provides support in unpackaged deployment mode.
 
+## Hot Reload Indicator
+
+Hot Reload also has a visual indicator to help you further monitor changes while developing. It displays new information every time Hot Reload is triggered. The indicator is enabled by default within the `EnableHotReload()` method which is located in the root App.xaml.cs file. This displays an overlay which hosts the indicator. If you wish to disable it, you simply have to provide the following boolean: `EnableHotReload(disableIndicator: true)`, removing the overlay from the view.
+
+<p align="center">
+  <img src="../Assets/features/hotreload/indicator.png" alt="HotReloadIndicator" />
+</p>
+
+> [!TIP]
+> The overlay is grabbable by the anchor on the left hand side and can be moved around so that it doesn't hide the view you are modifying.
+
+The indicator displays the current connection status. Clicking on it will open a flyout containing all events or changes that were made recorded by Hot Reload. These events display more details about Hot Reload changes, like its status and impacted files. Connection event statuses are also shown.
+
+<p align="center">
+  <img src="../Assets/features/hotreload/indicator-flyout.png" alt="HotReloadIndicatorFlyout" />
+</p>
+
+### Statuses
+
+Here's a summary of what icons and statuses you can expect:
+
+__Connection__:
+
+![HotReloadIconConnectionSuccess](../Assets/features/hotreload/status-connection-success.png) Success
+
+![HotReloadIconConnectionFailed](../Assets/features/hotreload/status-connection-failed.png) Failed (Hot Reload was not able to connect to the endpoint or lost connection)
+
+![HotReloadIconConnectionWarning](../Assets/features/hotreload/status-connection-warning.png) Warning (Version mismatch, unknown updates or processors missing)
```

Comment:
Wondering if we should repeat here content of messages to be more actionable. Maybe a simple message "this usually indicates an issue that can be solved by restarting your IDE".

Reply:
Good point, the icons speak for themselves in most cases, its better to have text that could help the user